inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -259,6 +260,14 @@ def _perform_cmd_type_operation(self, parsed_head: XmlHandler, xml: str) -> Resu
         elif cmd == "query":
             self._dispatcher_broker.mqtt_publish(QUERY_CMD_CHANNEL, xml)
             return PUBLISH_SUCCESS
+        elif cmd == "rpc":
```

Comment:
i suspect this may actually come in through `do_install` under `type_of_manifest == 'ota'` but am not sure so go ahead and try and see

Reply:
We discussed initially it in team would be rpc command as cmd just like query. So had that same format for this. Closing.